### ISSUE TITLE
Stripping php tags

### ISSUE THREAD

**Issue by lmartins** [Automattic/juice#57]
For some reason the plugin is producing undesired effects on PHP tags within the email:
```
<a href="<?php echo user()->getSystemAddress() ?>" target="_blank">
```
gets compiled to:
````
<a href="&lt;?php echo user()-&gt;getSystemAddress() ?&gt;" target="_blank" style="color: #348eda;">
`````

While others like:
````
<h3><?php echo $code ?></h3>
````
are compiled to:
````
<h3 style="font-family: Helvetica, Arial, sans-serif;"></h3>
````

Is there anything I can do to avoid this?

**Comment by levi-putna** [comment]
Do we have a fix for this issue?

**Comment by jrit** [comment]
@levi-putna this looks fixed in 1.0.0 - I just added a test case to cover examples in this issue. If there are still breaking cases, please post them so we can check if there is a resolution available.
